src/components/Queue.vue

```diff
@@ -12,7 +12,7 @@
                 {{ message }}
             </div>
             <div v-else class="message">
-                You have <strong>{{ vouchers.length }}</strong> vouchers in your queue.
+                [xXx]You have <strong>{{ vouchers.length }}</strong> vouchers in your queue. These voucher codes will be checked when you submit your queue.
```

Comment:
vuejs 2.0 removed filters like pluralize while on some "purity" purge. The docs (https://vuejs.org/v2/guide/migration.html) think https://www.npmjs.com/package/pluralize is a good replacement.